package.json

```diff
@@ -1,18 +1,20 @@
 {
-  "name": "codebuild-slack-notifier",
-  "version": "1.0.0",
-  "description": "Send CodeBuild events to Slack",
+  "name": "pipelines-slack-notifier",
```

Comment:
More than a minor change?

Reply:
my bad, shouldn't have committed this change